docs/framework/network-programming/automatic-proxy-detection.md

```diff
@@ -16,32 +16,30 @@ helpviewer_keywords:
   
-> [!NOTE]
-> You can display the Internet Explorer **Local Area Network (LAN) Settings** dialog box by selecting **Tools** from the Internet Explorer main menu and then selecting **Internet Options**. Next, select the **Connections** tab, and click **LAN Settings**.  
+ Automatic proxy detection is performed by the <xref:System.Net.WebProxy> class and can employ request-level settings, settings in configuration files, and computer proxy settings.
   
  When automatic proxy detection is enabled, the <xref:System.Net.WebProxy> class attempts to locate the proxy configuration script as follows:  
   
-1. The WinINet `InternetQueryOption` function is used to locate the proxy configuration script most recently detected by Internet Explorer.  
+1. The WinINet `InternetQueryOption` function is used to locate the proxy configuration script most recently detected by the browser.
   
 2. If the script is not located, the <xref:System.Net.WebProxy> class uses the Dynamic Host Configuration Protocol (DHCP) to locate the script. The DHCP server can respond either with the location (host name) of the script or with the full URL for the script.  
   
 3. If DHCP does not identify the WPAD host, DNS is queried for a host with WPAD as its name or alias.  
   
-4. If the host is not identified and the location of a proxy configuration script is specified by the Internet Explorer LAN settings or a configuration file, this location is used.  
+4. If the host is not identified and the location of a proxy configuration script is specified by the computer's proxy settings or a configuration file, this location is used.  
   
 > [!NOTE]
-> Applications running as an NT Service or as part of ASP.NET use the Internet Explorer proxy server settings (if available) of the invoking user. These settings may not be available for all service applications.  
+> Applications running as an NT Service or as part of ASP.NET use the computer's proxy server settings (if available) of the invoking user. These settings may not be available for all service applications.  
   
- Proxies are configured on a per-connectoid basis. A connectoid is an item in the network connection dialog, and can be a physical network device (a modem or Ethernet card) or a virtual interface (such as a VPN connection running over a network device). When a connectoid changes (for example, a wireless connection changes an access point, or a VPN is enabled), the proxy detection algorithm is run again.  
+ Proxies are configured on a per-connectoid basis. A connectoid is an item in the network connection dialog. It can be a physical network device, such as a modem or Ethernet card, or a virtual interface, such as a VPN connection that runs over a network device. When a connectoid changes, for example, a wireless connection changes an access point or a VPN is enabled, the proxy detection algorithm is run again.  
```

Comment:
```suggestion
 Proxies are configured on a per-connectoid basis. A connectoid is an item in the network connection dialog. It can be a physical network device, such as a modem or Ethernet card, or a virtual interface, such as a VPN connection that runs over a network device. When a connectoid changes, the proxy detection algorithm is run again. For example, proxy detection is run when a wireless connection changes an access point or a VPN is enabled.
```